selectors-4/Overview.bs

```diff
@@ -2403,23 +2411,63 @@ Resource State Pseudos</h2>
 	particularly images/videos,
 	and allow authors to select them based on some quality of their state.
 
-<h3 id="video-state">
-Video/Audio Play State: the '':playing'' and '':paused'' pseudo-classes</h3>
+<h3 id="video-state"> <!-- keeping this ID because Cool URIs Don't Change -->
+Media Playback State: the '':playing'', '':paused'', and '':seeking'' pseudo-classes</h3>
 
 	The <dfn>:playing</dfn> pseudo-class represents an element
-	representing an audio, video, or similar resource
 	that is capable of being “played” or “paused”,
 	when that element is “playing”.
 	(This includes both when the element is explicitly playing,
 	and when it's temporarily stopped for some reason not connected to user intent,
 	but will automatically resume when that reason is resolved,
-	such as a “buffering” state.)
+	such as a “buffering” or “stalled” state.)
 
-	The <dfn>:paused</dfn> pseudo-class represents the same elements,
-	but instead match when the element is <em>not</em> “playing”.
+	The <dfn>:paused</dfn> pseudo-class represents an element
+	that is capable of being “played” or “paused”,
+	when that element is “paused” (i.e. <em>not</em> ”playing”).
 	(This includes both an explicit “paused” state,
 	and other non-playing states like “loaded, hasn't been activated yet”, etc.)
 
+    The <dfn>:seeking</dfn> pseudo-class represents an element
```

Comment:
This spec is tab-indented, please stick with that consistently.

Reply:
Fixed in 6599d0b.